apps/desktop-wallet/src/features/assetsLists/NFTsGrid.tsx

```diff
@@ -53,18 +58,59 @@ interface NFTsGridProps {
   placeholderText: string
 }
 
-const NFTsGrid = ({ columns, nftIds, isLoading, placeholderText }: NFTsGridProps) => (
-  <motion.div {...fadeIn}>
-    {!isLoading && nftIds.length === 0 && <EmptyPlaceholder emoji="🖼️">{placeholderText}</EmptyPlaceholder>}
-
-    {isLoading ||
-      (nftIds.length > 0 && (
-        <Grid columns={columns}>
-          {isLoading ? <NFTsLoader /> : nftIds.map((nftId) => <NFTCard key={nftId} nftId={nftId} />)}
-        </Grid>
-      ))}
-  </motion.div>
-)
+const NFTsGrid = ({ columns, nftIds, isLoading, placeholderText }: NFTsGridProps) => {
+  const { data, fetchNextPage, hasNextPage } = useFetchPaginatedNFTs({ nftIds })
+
+  const ref = useRef(null)
```

Comment:
Nit: I find it useful to name the ref, especially when it doesn't reference the main container component. Here it could be `gridBottomRef`

Reply:
ok!